src/kudu/consensus/leader_election.h

```diff
@@ -61,7 +84,7 @@ class VoteCounter {
   // Otherwise, it is set to false.
   // If an OK status is not returned, the value in 'is_duplicate' is undefined.
   virtual Status RegisterVote(
-      const std::string& voter_uuid, ElectionVote vote, bool* is_duplicate);
+      const std::string& voter_uuid, VoteInfo vote_info, bool* is_duplicate);
```

Comment:
vote_info could be made a reference to avoid copies (and avoid my other suggestions of std::move())
```suggestion
      const std::string& voter_uuid, const VoteInfo& vote_info, bool* is_duplicate);
```

Reply:
will make it a reference